### ISSUE TITLE
UINavigationController's hidesBarsOnSwipe not working when the message is on screen

### ISSUE THREAD

**Issue by RomanPodymov** [SwiftKickMobile/SwiftMessages#190]
Hello.
Thank you for this library, I really like it.
However I found that when I set the property `hidesBarsOnSwipe` of `UINavigationController` to `true` navigation bar never appears after scrolling. Should I use some custom solution for navigation bar to be able to show/hide bar on scrolling when I am using SwiftMessages?

**Comment by wtmoose** [comment]
How did you conclude that SwiftMessages is breaking the behavior `hidesBarOnSwipe`? That doesn't seem possible to me.

**Comment by RomanPodymov** [comment]
Hello @wtmoose,
I made some changes now. In my controller I set another constraints for `UIScrollView`. Now it's leading, trailing, top and bottom are fixed to the root view, not to the safe area. And I guess it fixed the problem. However now the status bar is always visible, I need to hide it when the navigation bar is also hidden. In my class, derived from `UINavigationController` I have the following property:
```
override var prefersStatusBarHidden : Bool {

    return self.isNavigationBarHidden
}
```
Before I added SwiftMessages to my project it was OK. But now I don't know why the status bar is always visible when I show some message with SwiftMessages.
